.changeset/old-coats-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': major
+---
+
+Removes 'warning' status from form validation
```

Comment:
I wonder if it would be more explicit if we add `validationStatus` prop to the changeset text? Might be confusing maybe like what is `warning` status?